playbooks/webapp/tasks/create.yaml

```diff
@@ -43,6 +43,10 @@
       ansible.builtin.set_fact:
         shared_az: "{{ region_av_zones[0] }}"
 
+    - name: Print the value of Subnet_cidr
```

Comment:
I don't think this debug task is needed.

Reply:
made the change.  Thanks.